src/http_rest_api_handler.cpp

```diff
@@ -479,12 +483,12 @@ Status HttpRestApiHandler::processInferKFSRequest(const HttpRequestComponents& r
     timer.start("total");
     ServableMetricReporter* reporter = nullptr;
     std::string modelName(request_components.model_name);
-    std::string modelVersion(std::to_string(request_components.model_version.value_or(0)));
-    SPDLOG_DEBUG("Processing REST request for model: {}; version: {}", modelName, modelVersion);
+    std::string modelVersionLog = request_components.model_version.has_value() ? std::to_string(request_components.model_version.value()) : DEFAULT_VERSION;
```

Comment:
It should be passed into gRPC predict request as 0. Then it will be rejected since no such version is in OVMS.
If no version is specified then OVMS will choose default version.

Reply:
makes sense